src/WebJobs.Script/Scale/FunctionsScaleManager.cs

```diff
@@ -151,8 +150,18 @@ private async Task<IEnumerable<TargetScalerVote>> GetTargetScalersResultAsync(Sc
                                 _logger.LogDebug($"Snapshot dynamic concurrency for target scaler '{targetScaler.TargetScalerDescriptor.FunctionId}' is '{functionSnapshot.Concurrency}'");
                             }
                         }
-
-                        TargetScalerResult result = await targetScaler.GetScaleResultAsync(targetScaleStatusContext);
+                        TargetScalerResult result = null;
+                        try
+                        {
+                            result = await targetScaler.GetScaleResultAsync(targetScaleStatusContext);
+                        }
+                        catch (NotSupportedException ex)
+                        {
+                            string id = GetTargetScalerFunctionUniqueId(targetScaler);
+                            _logger.LogError($"Target scaler '{id}' is not supported, scale monitor will be used instead.", ex);
```

Comment:
Should this be a warning instead? If we end up auto-enabling for people, we might not want it to generate errors for a situation where we have a fallback.

Also, for the message, maybe "Unable to use target based scaling for Function '{functionName}'. Metrics monitoring will be used."